docker/core/scripts/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ssh root@167.99.88.190 "
```

Comment:
Replace it with `root@codesearch.aelve.com`

Reply:
Or actually – is this file needed at all here?

Reply:
You are right, i will clean other scripts. My fault forget to clean it before pushing.